libs/sdk-ui-kit/styles/scss/shareDialog.scss

```diff
@@ -283,37 +285,73 @@
         }
     }
 
-    &:hover {
+    &:hover,
+    &.is-active {
         background: variables.$is-focused-background;
 
         .gd-grantee-item-icon-right {
             display: block;
         }
     }
-}
 
-.gd-granular-permissions-dropdown {
```

Comment:
Removing this style could break plugins ... 
When plugin will start with fixed old version but this use current style this could do a mess for old plugin. Sorry about that ... this is known problem of css and plugin compatibility

Reply:
Until we enable the whole granular permissions flow, I believe we can do anything with these styles. So this is safe from my point of view as nobody is using these components yet.